app/admin/accounts/list/route.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model() {
+    return this.store.ajax({
+      url: '/api/v1.0/accounts'
+    }).catch(error => {
+      // handle not found error gracefully
+      var body = error.responseJSON;
+      if (body && body.code) {
+        return {
+          error: body
+        };
+      }
+      throw error;
+    });
+  },
+  setupController(controller, {model, error}) {
+    controller.setProperties({
```

Comment:
I don't think this will work if the ajax request is successful, as it won't return an object with a model property on it.  The fully resolved model will be passed in.

This will be passed in as the second argument:
```
[{name:'foo'},{name:'foo'}]
```

Not this:
```
{
  model: [{name:'foo'},{name:'foo'}]
}
```